readme.md

```diff
@@ -629,6 +629,7 @@
 - [Password Cracking](https://github.com/n0kovo/awesome-password-cracking#readme) - The process of recovering passwords from data that has been stored in or transmitted by a system in scrambled form.
 - [Security Card Games](https://github.com/Karneades/awesome-security-card-games#readme) - Train your skills and discuss various security topics.
 - [Suricata](https://github.com/satta/awesome-suricata#readme) - Intrusion detection/prevention system and network security monitoring engine.
+- [OpenID Connect](https://github.com/cerberauth/awesome-openidconnect#readme) - OpenID Connect providers, services, libraries and resources.
```

Comment:
> It should not describe the list itself. - guidelines

Reply:
Thanks for your review.

The description is updated. It now describes OpenID Connect itself.